pinot-connectors/prestodb-pinot-dependencies/presto-pinot-driver/src/main/java/org/apache/pinot/connector/presto/PinotScatterGatherQueryClient.java

```diff
@@ -196,10 +197,12 @@ public PinotScatterGatherQueryClient(Config pinotConfig) {
     _brokerMetrics = new BrokerMetrics(PinotMetricUtils.getPinotMetricsRegistry());
     _brokerMetrics.initializeGlobalMeters();
     TlsConfig tlsConfig = getTlsConfig(pinotConfig);
+    ServerRoutingStatsManager serverRoutingStatsManager = new ServerRoutingStatsManager(new PinotConfiguration());
```

Comment:
Same here. Can we avoid changing such code where this feature will never be used ?

Reply:
If we pass null, we will have to perform a if/else null check everywhere serverRoutingStatsManager is used. Leaving as such based on offline discussion.